src/components/datatable/DocumentsTable.vue

```diff
@@ -46,6 +48,25 @@
     return Object.assign(result, options);
   }
 
+  // comparator functions for custom sort of "complicated" columns
+  // these functions could be removed if the sort function of
+  // AG grid could be deactivated since documents are expected to
+  // be given in correct order by the backend
+  function condComp(cond1, cond2) {
```

Comment:
Please use as possible explicit names (and blame me when I didn't do this :) ) 

```suggestion
  function conditionComparator(cond1, cond2) {
```